### PR TITLE
Added generic physical node actors macros and pages

### PR DESCRIPTION
**Pull request by abdulrahmantkhalifa**
- replaced the stacks and storage routers pages with physical node and nodes pages 
 - added the enable and maintenance end point for generic nodes

### CODE REVIEW

**Comment by abdulrahmantkhalifa** [comment]
WIP, NOT DONE

**Comment by abdulrahmantkhalifa** [comment]
done

File: apps/cbportal/base/CBGrid/.macros/wiki/physical_node_data/1_physical_node_data.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        args.doc.applyTemplate(data, True)
+        return params
+    data['node'] = node
+
+    for netaddr in node['netaddr']:
+        if netaddr['name'] == 'storage':
+            data['storageip'] = netaddr['ip'][0]
+            break
+    else:
+        raise RuntimeError('Storage Node does not have interface for storage ip , please specify storage interface')
```
Comment (grimpy) [inline apps/cbportal/base/CBGrid/.macros/wiki/physical_node_data/1_physical_node_data.py]: no runtime errors in macros, instead make it print something on the page something usefull

**Comment by abdulrahmantkhalifa** [comment]
relates to https://github.com/0-complexity/openvcloud/issues/1433

File: apps/cbportal/base/cloudbroker__ovsnode/methodclass/cloudbroker_ovsnode.py [hunk 2]
```diff
@@ -52,12 +63,12 @@ def get_vpool_name(host, port):
             return None
 
         def get_vdisk(vdiskguid):
-            disks = self.models.disk.search({'referenceId': {'$regex': vdiskguid}})[1:]
-            if disks:
-                return disks[0]
+            disk = self.models.disk.searchOne({'referenceId': {'$regex': vdiskguid}})
```
Comment (grimpy) [inline apps/cbportal/base/cloudbroker__ovsnode/methodclass/cloudbroker_ovsnode.py]: just return

**Pull request merged**